gravitee-apim-console-webui/src/management/api/creation/v4/steps/step-6/api-creation-v4-step-6.component.scss

```diff
@@ -2,6 +2,38 @@
 @use '@gravitee/ui-particles-angular' as gio;
 
 .api-creation-v4-step-6 {
+  &__body {
+  }
+  &__step {
+    display: flex;
+    border: 1px solid mat.get-color-from-palette(gio.$mat-neutral-palette, 'default');
+    border-radius: 6px;
+    padding: 16px;
+    gap: 30px;
+
+    &__info {
+      display: flex;
+      flex-direction: column;
+      flex: 1 1 100%;
+
+      &__value {
+        font-weight: 700;
+        color: mat.get-color-from-palette(gio.$mat-accent-palette, 'darker20');
+        margin-bottom: 8px;
+      }
+      &__row {
+        display: flex;
+        gap: 4px;
+      }
+    }
+
+    &__button {
+      margin-top: 90px;
+      margin-bottom: auto;
+      width: 85px;
```

Comment:
How about letting the button take the size it needs ? 
Could we try to replace these lines and use the parent flex to align vertically?